db/migrate/20150830044919_create_participants.rb

```diff
@@ -0,0 +1,11 @@
+class CreateParticipants < ActiveRecord::Migration
+  def change
+    create_table(:participants, { id: :uuid }) do |t|
+      t.uuid(:joinable_id, { null: false })
+      t.string(:joinable_type, { null: false })
```

Comment:
The idea behind going polymorphic here is to be able to add people to Missions or Tasks via the same table -- can anyone think of a reason why we may want to store different information across those types of joins? The only thing I thought of is a Mission may have a user that's an "owner", but that could be captured by a generic "creator" field that would apply to both kinds.